app/views/govuk_publishing_components/components/docs/checkbox.yml

```diff
@@ -0,0 +1,39 @@
+name: Checkbox
```

Comment:
This is called `Checkbox` but the other component is `Form checkboxes`. Would be better if they were both called something similar, if only so they appear next to each other in the list of components.